prow/config.yaml

```diff
@@ -50,81 +50,22 @@ github_reporter:
   - presubmit
   - postsubmit
 
-blocked_branches: &blocked_branches
-  release-*: &blocking_merge
-    required_status_checks:
-      contexts:
-      - "merges-blocked-needs-admin"
-    protect: true
-
 branch-protection:
-  allow_disabled_policies: true
+  protect: true
```

Comment:
This will break how our community uses branches. There are non-standard branches in  our repos which should not have branch protection enabled. We only want master and release-* to have branch protection.

Reply:
1) What does this help? What does it accomplish that they cannot do with a fork?
2) If these are useful why shouldn't they have branch protection?
3) If they are small in number, we can specifically exclude them (like we do in kubernetes)?